bots/tests-invoke

```diff
@@ -373,8 +373,13 @@ class PullTask(object):
 
 
         env = os.environ.copy()
-        if "TEST_DATA" in env:
-            del env["TEST_DATA"]
+        # we can only use the cache for master; branches still use the old
+        # test/images/ symlink farm which we must not destroy
+        if opts.rebase:
+            if "TEST_DATA" in env:
+                del env["TEST_DATA"]
+        else:
+            env["TEST_DATA"] = BOTS
```

Comment:
Why do we set this to BOTS? It should just stay set to whatever it was in this case.